x-pack/plugins/security_solution/public/detections/components/alerts_table/default_config.tsx

```diff
@@ -122,6 +122,34 @@ export const buildAlertsFilter = (ruleStaticId: string | null): Filter[] =>
       ]
     : [];
 
+export const buildAlertsFilterByRuleIds = (ruleIds: string[] | null): Filter[] => {
```

Comment:
`ruleStaticIds` is `string[]` in the current usage, so why do we handle `| null` here (and why is it `null` instead of `undefined`)?

Edit: I see that the original `buildAlertsFilter` accepts `null` as well, so I'm guessing that's where that behavior came from? I don't see any example of actually needing to handle that case, though.